MANIFEST.in

```diff
@@ -1 +1 @@
-include README.md LICENSE
+include README.md LICENSE test/conftest.py
```

Comment:
We could also use `recursive-include test *.py` to include all Python files from `test/`.

Reply:
+1